Library/Extensions/Feature+Helpers.swift

```diff
@@ -7,6 +7,7 @@ public func featureNativeCheckoutIsEnabled() -> Bool {
 
 public func featureNativeCheckoutPledgeViewIsEnabled() -> Bool {
```

Comment:
I feel like a `guard` more clearly explains the 'gate-keeping' nature of what we want to do here:

```swift
public func featureNativeCheckoutPledgeViewIsEnabled() -> Bool {
  // native checkout pledge view is currently not available in release builds
  guard !AppEnvironment.current.mainBundle.isRelease else { return false }

  return Feature.nativeCheckoutPledgeView.isEnabled()
}
```